### PR TITLE
#3 readme blurb

### PR DESCRIPTION
**Pull request by jthomasw88**

### CODE REVIEW

**Review by jadavey91 — DISMISSED** [review]
Just don't let them probe you!

**Review by boothscript — APPROVED** [review]
Stephen Booth approves this message

File: jthomasw88/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+my name is Jordan WIlliams, I dont know where I am & I how I got here, I think I have possibly been taken hostage. Please send help!!
```
Comment (nmunro) [inline jthomasw88/README.md:1]: Surely git isn't THAT bad...

**Review by nmunro — APPROVED** [review]

**Review by boothscript — APPROVED** [review]

**Review by jadavey91 — APPROVED** [review]
I keep forgetting to tick Approve, argh!

**Review by phiddle — APPROVED** [review]
looks good. line 32 the < is that needed ?

**Pull request merged**